roles/provision-keycloak-apb/tasks/addrealmuser-keycloak.yml

```diff
@@ -0,0 +1,127 @@
+---
+- name: Generate KC development username
+  shell: tr -d -c "a-zA-Z" < /dev/urandom | head -c 20
+  register: kcdev_username
```

Comment:
do we want to do this or just create a user with the same name as the namespace so it is obvious which user is for which namespace etc

Reply:
Done!